runtime/commonMain/src/kotlinx/serialization/descriptors/SerialKinds.kt

```diff
@@ -2,10 +2,10 @@
  * Copyright 2017-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
 
-package kotlinx.serialization
+package kotlinx.serialization.descriptors
```

Comment:
Shouldn't we rename this file to `SerialKind.kt`?

Reply:
Not so sure about that. The root level class is SerialKind, but we have many kinds in it. Also, it does not have any extensions, so `SerialKindsKt` is not present in the ABI.